README.md

```diff
@@ -5,6 +5,17 @@ Projects are created for each topic in [Freecodecamp](https://www.freecodecamp.o
 
 Visit [trello board](https://trello.com/b/XGKSW6rb/nathaly-learning-path) for project status and progress.
 
+## Prerequisites
+
+- [NodeJS](https://nodejs.org/en/download).
```

Comment:
We need to ensure these extensions are installed in VS Code so that the linters will work correctly.